### DISCUSSION TITLE
Unable to Drag images into fooocus

### DISCUSSION THREAD

**Discussion by SkelegonDK** [lllyasviel/Fooocus#3272] [Q&A] [answered]
OS: MacOS --latest
Running on: Google Colab

I'm still not able to drag images from the finder to fooocus on the browser, neither can I generated images directly from one window of fooocus to another. Probably mac specific as I haven't see anyone else have this issue.

**Comment by mashb1t** [comment, marked as answer]
Drag & Drop from generated image to any input image only works when running locally (not in Colab / any servr) and when using Chrome.

**Reply by Moorer624** [reply]
I'm on a Mac M1Max and it has never worked for me - with Chrome, Safari, any browser. I run Fooocus locally. Any thoughts?

**Comment by SkelegonDK** [comment]
I did some digging after the fact, sorry to bother you. Thanks for answering anyway.
